deploy/gk-deploy

```diff
@@ -148,25 +113,6 @@ abort() {
   exit 1
 }
 
-assign() {
-  key="${1}"
-  assign=`expr index ${key} '='
-  `
-  if [[ $assign -gt 0 ]]; then
-    echo "${key:assign}"
-    return 0
-  elif [[ $assign -eq 0 ]] &&
-       [[ "x${2}" != "x" ]]; then
-    echo "${2}"
-    return 2
-  else
-    output "Required parameter for '-${key}' not specified.\n"
-    usage
-    exit 1
-  fi
-  keypos=$keylen
-}
-
 check_pods() {
```

Comment:
It depends on having $CLI defined, so I'd say no. Plus I haven't found a need for it yet.